### PR TITLE
Update to new table of contents structure

### PR DESCRIPTION
**Pull request by willingc**
Addresses #67

### CODE REVIEW

File: doc/source/cost.rst [hunk 1]
```diff
@@ -56,6 +56,6 @@ Data 8
 The Data 8 course at UC Berkeley used a JupyterHub to coordinate all course
 material and to provide a platform where students would run their code. This
 consisted of many hundreds of students, who had minimal requirements in terms
-of CPU and memory usage. Ryan Lovett put together
-`a short jupyter notebook <https://github.com/data-8/jupyterhub-k8s/blob/master/docs/cost-estimation/gce_budgeting.ipynb>`_
-estimating the cost for computational resources depending on the student needs.
+of CPU and memory usage. Ryan Lovett put together a short `Jupyter notebook
```
Comment (choldgraf) [inline doc/source/cost.rst]: at some point I should try finalizing this one: http://predictablynoisy.com/HubCost/costs.html

File: doc/source/index.rst [hunk 2]
```diff
@@ -16,24 +16,50 @@ questions, feel free to reach out at the `issues page`_.
...
-    setup-jupyterhub
-    turn-off
+   getting-started
+   create-k8s-cluster
+   setup-helm
+   setup-jupyterhub
+   turn-off
 
-**Extra info and tips**
+**Customization Guide**
```
Comment (choldgraf) [inline doc/source/index.rst]: this is a great idea to break it down by topic like this

**Comment by choldgraf** [comment]
I like it! Nice job to the pycon team for improving the docs :)

**Comment by willingc** [comment]
Thanks @choldgraf. I'm hoping we get a chance this week to work on in person.

**Comment by choldgraf** [comment]
oh that's right! I forgot you all are visiting! I'm back in town starting Wednesday so would love to work on this then!

**Pull request merged**